docker/carbon-black-cloud/Pipfile

```diff
@@ -7,6 +7,7 @@ name = "pypi"
 carbon-black-cloud-sdk = "*"
 
 [dev-packages]
+cbapi = "*"
```

Comment:
@esharf @ilappe I think this was added at the wrong place. Under dev-packages instead of packages. Please check.

Reply:
already fixed in #5041 
Thanks